operator/k8s_cep_gc.go

```diff
@@ -67,6 +67,7 @@ func enableCiliumEndpointSyncGC(once bool) {
 		// state.
 		watchers.PodsInit(k8s.WatcherCli(), stopCh)
 	}
+	<-k8sCiliumNodesCacheSynced
```

Comment:
In `onOperatorStartLeading()` we make a similar call, but restricted to `ipamMode` == `ipamOption.IPAMOperator`. Is this OK to unconditionally call here?

Reply:
Good question, I'll have to make sure the node watcher is started in all cases.

Reply:
Adding a dummy Node Manager to be used when IPAM mode needs no help from the Operator (e.g, `IPAMCRD`).